invenio_search/config.py

```diff
@@ -94,3 +94,6 @@
     "980__b": ["collections.secondary"],
     "542__l": ["information_relating_to_copyright_status.copyright_status"],
 }
+
+SEARCH_ELASTICSEARCH_RECORDS_DOC_TYPE = 'record'
```

Comment:
`SEARCH_DOCTYPE_DEFAULT` is not too generic? In this case the configuration  is something strictly correlated with `record`.